### ISSUE TITLE
Parallel-RDP and RSP Not Work

### ISSUE THREAD

**Issue by ToniBC** [libretro/mupen64plus-libretro-nx#265]
Now when selecting the parallel RDP & RSP plugin, the core does not load, Retroarch closes.

Here I leave the log.

[retroarch.log](https://github.com/libretro/mupen64plus-libretro-nx/files/5305137/retroarch.log)

Thank you.

**Comment by m4xw** [comment]
![image](https://user-images.githubusercontent.com/13141469/94683971-a4e4d480-0327-11eb-9f0c-e2b874c82edb.png)
Works just fine (build from the other issue).
Make sure you set your driver to "vulkan" and that your graphics drivers are up2date

**Comment by ToniBC** [comment]
The same as with the other error @m4xw , it does not work, Windows is updated, the nvidia drivers in their latest version 456.55 from 2 days ago. In the log I think everything comes out, it is the only thing that retroarch records in debug mode.

Tested in Retroarch clean both Stable and Nightly, just download the core and load, the overscan does not work and before if, and with the drivers parallel neither load, Retroarch closes.

They notified me of that error, because they were trying to adjust the image and it was not working for them and I checked it and it does not work.

I even tried that version you left, which is the same, but to be tested.

Tried with GL and Vulkan and nothing.
